### ISSUE TITLE
Question about loss function during segmentation

### ISSUE THREAD

**Issue by LW130003** [xu-ji/IIC#98]
Hi Xu-Ji,

Currently, I have modified the Segmentation Script to train another dataset. Both the dataset loader B and A are the same but I noticed the avg loss returned (between head B and A) have large margin of difference:
![image](https://user-images.githubusercontent.com/26461142/103719706-8ff19e80-5004-11eb-96d5-40aacc4526fb.png)

Any idea why this happened? Am I training it correctly

Thank you and regards

**Comment by xu-ji** [comment]
If they have different numbers of clusters, the losses for the two heads will likely be different, so I do not think it is cause for concern.
